src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Completion/Extension/ICompletionExtension.cs

```diff
@@ -0,0 +1,40 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.ServiceLayer.Connection;
+using Microsoft.SqlTools.ServiceLayer.LanguageServices.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SqlTools.ServiceLayer.LanguageServices.Completion.Extension
+{
+    public interface ICompletionExtension : IDisposable
+    {
+        /// <summary>
+        /// Unique name for the extension
+        /// </summary>
+        string Name { get; }
+
+        /// <summary>
+        /// Completion extension initialization
```

Comment:
>Completion extension initialization [](start = 12, length = 35)

I'd suggest something more useful like "Method for initializing the extension, this is called once when the extension is loaded"